packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts

```diff
@@ -146,6 +146,10 @@ export class CliArgsHandlerBase implements ICliArgsHandler {
           type: 'boolean',
           describe: 'If the cache should be disabled',
         },
+        distinct: {
```

Comment:
Since this only applies to CONSTRUCT queries, maybe we'll want to rename it to `distinctConstruct` in all locations?